specification/columns/availabilityzone.md

```diff
@@ -2,7 +2,7 @@
 
 An [*availability zone*](#glossary:availability-zone) is a provider-assigned identifier for a physically separated and isolated area within a Region that provides high availability and fault tolerance. Availability Zone is commonly used for scenarios like analyzing cross-zone data transfer usage and the corresponding cost based on where [*resources*](#glossary:resource) are deployed.
 
-The AvailabilityZone column is RECOMMENDED to be present in the billing data when the provider supports deploying resources or services within an *availability zone*. This column MUST be of type String and MAY contain null values when a charge is not specific to an *availability zone*.
+The AvailabilityZone column is RECOMMENDED to be present in a FOCUS dataset when the provider supports deploying resources or services within an *availability zone*. This column MUST be of type String and MAY contain null values when a charge is not specific to an *availability zone*.
```

Comment:
I assume we need to add links to the term in the glossary.

Not sure if we should use `<a name="glossary:FOCUS-dataset"><b>FOCUS Dataset</b></a>` or `<a name="glossary:focus-dataset"><b>FOCUS Dataset</b></a>`. For comparison, we've used `<a name="glossary:sku"><b>SKU</b></a>`, If the link/syntax works, I guess it doesn't matter.

```suggestion
The AvailabilityZone column is RECOMMENDED to be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports deploying resources or services within an *availability zone*. This column MUST be of type String and MAY contain null values when a charge is not specific to an *availability zone*.
```

Reply:
btw, @aqu-erp great job!

Reply:
Great catch!